CHANGELOG.md

```diff
@@ -1,3 +1,92 @@
+# v0.41.0 Release - 09/26/2019
+
+*This command has been implemented for the `kubectl` deployer only; implementations for `kustomize` and `helm` will follow in the next release.*
+
+*Note*: This release also comes with a new config version `v1beta17`.
+        To upgrade your `skaffold.yaml`, use `skaffold fix`. If you don't upgrade, skaffold will auto-upgrade in memory as best it can, and print a warning message.
+        See [Skaffold Deprecation Policy](http://skaffold.dev/docs/references/deprecation/) for details on what beta means.
+
+*Note*: 
```

Comment:
remove

Reply:
ah, forgot to add the note for deprecation for custom builder $IMAGES, will add, thanks